inputs/syntax_inputs.txt

```diff
@@ -1,22 +1,26 @@
 #This file describes the functionality of inputs within rSFSTEP2 and the options that exist for specifying inputs
 
-Three input files are held in the inputs folder:
+Four input files are held in the inputs folder:
 1) InputData_Rgroup.csv --> contains rgroup parameters, along with parameters for fire and grazing (top table of rgroup.in only)
 
 2) InputData_SoilLayers.csv --> contains soil depth and soil properties 
 
 3) InputData_Species.csv --> contains species parameters (top table of species.in only)
 
+4) InputData_GrowingSeason --> defines the predicted growing seasons for each functional group.
+
```

Comment:
contains the growing season for each functional group, along with phenological activity values, biomass, litter, and % live values (held in sxw_phen.in and sxw_prod.v2.in)

Reply:
Resolved by commit f15f202.